build/helm/edgemesh/values.yaml

```diff
@@ -31,5 +31,6 @@ agent:
   modules:
     edgeProxy:
       enable: true
+      invertServiceFilter: false
```

Comment:
If this is a non-required parameter, do not configure it.

Reply:
removed.